index.html

```diff
@@ -4063,20 +4063,18 @@ <h2>Elements</h2>
 <p>An ECMAScript <a>Object</a> <dfn>represents a web element</dfn>
  if it has a <a>web element identifier</a> <a>own property</a>.
 
-<p>Each <a>browsing context</a> has an associated <dfn>list of
- known elements</dfn>.
- When the <a>browsing context</a> is <a>discarded</a>,
- the <a>list of known elements</a> is discarded along with it.
```

Comment:
I wonder if we should keep this sentence but updated to match the appropriate window.

Reply:
I think it's unnecessary; it should be obvious that something owned by the Window has the same lifecycle as the Window object.